src/main/java/com/meilisearch/sdk/model/SearchResult.java

```diff
@@ -0,0 +1,36 @@
+package com.meilisearch.sdk.model;
+
+import lombok.Getter;
+import lombok.ToString;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * Result of `search` API
+ */
+@ToString
+public class SearchResult implements Serializable {
+
+	@Getter
+	ArrayList<HashMap<String, Object>> hits;
+
+	@Getter
+	int offset;
+
+	@Getter
+	int limit;
+
+	@Getter
+	int nbHits;
+
+	@Getter
+	boolean exhaustiveNbHits;
+
+	@Getter
+	int processingTimeMs;
+
+	@Getter
+	String query;
+}
```

Comment:
Sorry if I miss something, but I think `exhaustiveFacetsCount` and `facetsDistribution` are missing and are really useful 😊

Reply:
@curquiza thanks~
Also, how do you think about name? Is `SearchResult` okay, or how about `SearchResponse`?
Please let me know if you have suggestion...

Reply:
I would rather `SearchResult` 🙂 
"Response" refers more to the raw body returned by MeiliSearch, and not the class, from my POV.